internal/server/iigo.go

```diff
@@ -13,7 +13,7 @@ func (s *SOMASServer) runIIGO() error {
 
 	nonDead := getNonDeadClientIDs(s.gameState.ClientInfos)
 	updateAliveIslands(nonDead)
-	iigoSuccessful, iigoStatus := iigointernal.RunIIGO(&s.gameState, &s.clientMap)
+	iigoSuccessful, iigoStatus := iigointernal.RunIIGO(s.iigoServer, &s.gameState, &s.clientMap)
```

Comment:
you can just pass in `s.Logf` i think

Reply:
oh so no need to make the iigoserver struct ? lol

Reply:
or do you mean s.iigoServer.logf ?

Reply:
`s.logf` is a logger function you can pass in